action.yml

```diff
@@ -13,6 +13,9 @@ inputs:
   password:
     description: (Optional) Docker's registry password (needed to push image to repository, or to pull from private repository)
     required: false
+  session:
+    description: (Optional) Docker's registry session (needed to push image to repository, or to pull from private repository)
```

Comment:
I am not sure if this is the right input name and description. I would just name it  `aws_session_token` as it is specific to aws.

Reply:
didn't use AWS prefix as I thought the idea was to make it generic.
Still prefer such a specific name?

Reply:
I prefer generic if I think it might be used somewhere else, but in this case it would generate confusion IMO. So yes, in this case I prefer the specific name, unless you can think of a scenario where `session` might be used beyond AWS.
Thanks.

Reply:
@whoan I can totally imagine that other providers might have a third parameter for the auth

Reply:
Sounds good @bubenkoff. Could you please change the description to something like:

    (Optional) Docker's registry session (extra auth parameters)

Thanks for the contribution!

Reply:
done

Reply:
added both to readme and actions.yml